pages/bookmarks.jsx

```diff
@@ -30,8 +30,6 @@ export default React.createClass({
       if (pageSettings.shouldRestore) {
         // restore state back to what is stored in pageSettings
         this.setState(pageSettings.currentList);
-      } else {
```

Comment:
this object has no `fetchData` function - instead it should just do "whatever the code does normally" when not explicitly instructed to use the cached results.